2022-07_sprint/sprint_parties/index.md

```diff
@@ -1,8 +1,185 @@
 # Sprint parties
 ## Organizers
+### Data Umbrella Organizers
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://twitter.com/BerylKanali
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Beryl Kanali" src="../../_static/beryl.JPG" class="rounded-circle">
+
+Beryl Kanali
+:::
+:::{grid-item-card}
+:link: https://github.com/symeneses
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Sandra Meneses" src="../../_static/sandra.jpg" class="rounded-circle">
+
+Sandra Meneses
+:::
+:::{grid-item-card}
+:link: https://www.linkedin.com/in/sandy-weng-a0959762/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Sandy Weng" src="../../_static/sandy.jpg" class="rounded-circle">
+
+Sandy Weng
+:::
+::::
+
+### PyMC Organizers
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: TODO
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Christian Luhmann" class="rounded-circle" src="../../_static/christian.jpg">
+
+Christian Luhmann
+:::
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://oriolabrilpla.cat
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Oriol Abril Pla" src="../../_static/oriol.jpg" class="rounded-circle">
+
+Oriol Abril Pla
+:::
+:::{grid-item-card}
+:link: https://twiecki.io/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Thomas Wiecki" src="../../_static/thomas.jpg" class="rounded-circle">
+
+Thomas Wiecki
```

Comment:
@twiecki we need an image and double checking the link is your preferred one.

Reply:
@OriolAbril 
Let's use this one for @twiecki
primary link:  https://www.pymc-labs.io/team/thomas-wiecki/

I am submitting a PR with photos now.